### DISCUSSION TITLE
Feedback For Beta Projects: Formatting Codes

### DISCUSSION THREAD

**Discussion by northwesttreesgaming** [community/community#7054] [Projects and Issues]
I thought I would pitch in and provide some feedback. After using your beta Projects option I like how its structure.
I did notice that formatting using the bullet points check boxes, and headers are not supported or at least not at the moment.
I use boards for planning what I am doing with my projects and I would hate to see support dropped for them.

Anyhow I just thought I would pitch in and provide my feedback. I hope the support is dropped in the future.

**Comment by Katsute** [comment]
related: <https://github.com/github/feedback/discussions/4422>
